e2e/tests/nightly/genomic-extraction-to-vcf.spec.ts

```diff
@@ -99,6 +99,10 @@ describe('Genomics Extraction Test', () => {
     // Change Compute Type to Dataproc Cluster.
     await runtimePanel.pickComputeType(ComputeType.Dataproc);
 
+    // Increase runtime auto-pause time because runtime will auto. pause after 30 min (default value) of idle
+    // while export to vcf file still in progress.
+    await runtimePanel.pickAutoPauseTime(AutoPauseIdleTime.EightHours);
```

Comment:
Could you please also restore this or delete the runtime at the end of the test? This will increase costs otherwise, espeically if other workspaces pick up this runtime setting.

Reply:
Yes. Line #196 - #201 in this test are deleting runtime.
I thought it was strange there isn't a `1 hour` option. `8 hours` was too long but it is the next choice that's bigger than `30 minutes (default)`.

Reply:
<img width="316" alt="Screen Shot 2021-10-05 at 1 43 01 PM" src="https://user-images.githubusercontent.com/35533885/136075268-2a0eadbe-9dbc-4eab-96e9-4ea685493687.png">

Maybe it should be a separate ticket to add a smaller value.

Reply:
Thanks, I missed that.

Yes, we might need an intermediate value. I think Shimon is hoping to get feedback from demo users and tune this accordingly.